openapi.yaml

```diff
@@ -0,0 +1,139 @@
+    name: GNU Affero General Public License Version 3
+    url: http://www.gnu.org/licenses/agpl-3.0.en.html
+  contact:
+    name: IS Data Architecture Team
+    url: https://is.oregonstate.edu/data-architecture
+    email: isdataarchitecture@oregonstate.edu
+externalDocs:
+  description: GitHub Repository
+  url: https://github.com/osu-mist/express-api-skeleton
+servers:
+  - url: https://oregonstateuniversity-dev.apigee.net/v1
+    description: Development server
+  - url: https://api.oregonstate.edu/v1
+    description: Produciton server
+paths:
+  /express-api-skeleton:
+    get:
+      summary: List all APIs
+      description: Get all APIs
+      operationId: getApis
+      tags:
+        - apis
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Resources'
+        500:
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Errors'
+  /express-api-skeleton/{id}:
+    get:
+      summary: Info for a specific API
+      description: Get API by unique ID
+      operationId: getApiById
+      tags:
+        - apis
+      parameters:
+        - name: id
+          in: path
+          description: Unique API ID
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Resource'
+        404:
+          description: Resource not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Errors'
+        500:
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Errors'
+components:
+  schemas:
+    Links:
+      properties:
+        self:
+          type: string
+          format: url
+          description: Self link of resource
+    Resource:
+      properties:
+        id:
+          type: integer
+          format: int64
+          description: A unique ID of API
+        type:
+          type: string
+          example: apis
+        links:
+          $ref: '#/components/schemas/Links'
+        attributes:
+          properties:
+            name:
+              type: string
+              description: Name of API
+              example: Location Frontend API
+            repoUrl:
+              type: string
+              format: url
+              description: Repository URL of API
+              example: https://github.com/osu-mist/locations-frontend-api
+    Resources:
+      properties:
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/Resource'
+    Error:
```

Comment:
Can you add a `code` field so it matches our dropwizard api skeleton error object? https://github.com/osu-mist/web-api-skeleton/blob/81758f9df25bf8003a3f09efbc5d6c9390b36714/swagger.yaml#L61-L79